contrib/cirrus/setup_environment.sh

```diff
@@ -364,8 +364,9 @@ case "$TEST_FLAVOR" in
         pip install --requirement $GOSRC/test/apiv2/python/requirements.txt
         ;&  # continue with next item
     compose)
-        make install.tools
-        dnf install -y podman-docker*
+        showrun make install.tools
+        showrun dnf remove -y gvisor-tap-vsock
+        showrun dnf install -y podman-docker*
```

Comment:
@lsm5, @containers/podman-maintainers this was intended to fix the CI breakage, [and it did](https://cirrus-ci.com/task/5820710329778176?logs=setup#L604), but I don't know how/if this will work after @lsm5's `Obsoletes` fix, or on next CI-VM update, or when the moon wanes gibbous.